src/app/static/src/app/components/plots/bubble/SizeLegend.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+    <l-control position="bottomleft">
+        <div class="map-control p-1">
+            <svg :width="width" :height="height">
+                <circle v-for="(circle, index) in circles" :key="'circle-' + index" stroke="#aaa" stroke-width="1" fill-opacity="0"
+                        :r="circle.radius" :cx="circle.x" :cy="circle.y"></circle>
+                <text v-for="(circle, index) in circles" :key="'text-' + index" text-anchor="middle"
+                      :x="circle.textX" :y="circle.textY">{{circle.text}}</text>
+            </svg>
+        </div>
+    </l-control>
+</template>
+
+<script lang="ts">
+    import Vue from "vue";
+    import {LControl} from "vue2-leaflet";
+    import {getRadius} from "./utils";
+    import {NumericRange} from "../../../types";
+
+    interface Circle {
+        x: number
+        y: number,
+        radius: number,
+        text: string,
+        textX: number,
+        textY: number
+    }
+
+    interface Props {
+        indicatorRange: NumericRange,
+        minRadius: number,
+        maxRadius: number
+    }
+
+    interface Computed {
+        circles: Circle[],
+        width: number,
+        height: number
+    }
+
+    const numeral = require('numeral');
+
+    export default Vue.extend<{}, {}, Computed, Props>({
+        name: "SizeLegend",
+        props: {
+            "indicatorRange": Object,
+            "minRadius": Number,
+            "maxRadius": Number
+        },
+        components: {
+            LControl
+        },
+        computed: {
+            width: function() {
+                return (this.maxRadius * 2) + 2; //leave room for stroke width
+            },
+            height: function() {
+                return (this.maxRadius * 2) + 10; //leave room for the top text
+            },
+            circles: function() {
+                const x = this.width / 2;
+                const circleFromRadius = (r: number, value: number, under: boolean = false) => {
+                    const y = this.height - r;
+
+                    let text = value > 1000 ? numeral(value).format("0a") : (+value.toFixed(2)).toString();
```

Comment:
I think we could do with more precision here - probably 3 dp for small numbers and `0.0a` format for large?
E.g. makes the smallest circle here <0.004 rather than literally 0:

![prevalencecircles](https://user-images.githubusercontent.com/6736820/72155819-dee20500-33ab-11ea-80d6-0fbce9ddfb0b.png)
 vs

![Screenshot from 2020-01-10 13-23-57](https://user-images.githubusercontent.com/6736820/72156275-0a192400-33ad-11ea-8b97-4f55f040e3e8.png)

and < 5k becomes < 4.6k here:

![Screenshot from 2020-01-10 13-26-35](https://user-images.githubusercontent.com/6736820/72156209-e2c25700-33ac-11ea-8573-f049b1e87418.png)

The tooltips actually have 4 dp but that might be too cluttered for the legend. At the moment the big numbers in the tooltips aren't formatted, we should open a ticket to format them to 0.0a also.

Reply:
OK, that sounds good. Do you think we should also have a couple of levels of 'smallness'? Small currently means anything under 100, so we'd potentially get '8647.1522' etc 
Maybe make it 4 sig figs rather than 4dps?

Reply:
4 sig fig sounds good to me, want to change that over?